A lot of bug fixes and new features!

Norbiros opened the pull request:
## Changes
- Fix bug with variables and lists in show block
- Escape HTML by default (you can show unescaped text with {{show unescaped {_un}}})
- Information about config updates (@SkyCraft78 maybe you know how we can update config?)
- New expression `cookie "__Host-ID" of event-request`
- Fix grammar and comments in `config.yml`
- New options in config (path, protected files, default value, new lines parsing)
- Add UTF-8 by default

## To Do
- [ ] More work on template system
- [ ] Fix encoding
- [ ] WebSK config reloading

Norbiros committed:
New expression - request cookie!

Norbiros committed:
Variable bug fixes + new features!

Norbiros committed:
Remove debugging messages

Norbiros committed:
Fix utf-8 with images

Norbiros committed:
Fix utf-8 with images

ItsTheSky commented:
Like reloading the webSK configuration?

Norbiros commented:
Great idea!

Norbiros committed:
Merge branch 'jensjeflensje:main' into bug-fixes-and-new-features

ItsTheSky commented:
No but it was a question about that 😂 
> Information about config updates (@SkyCraft78 maybe you know how we can update config?)

Norbiros commented:
> No but it was a question about that joy
> 
> > Information about config updates (@SkyCraft78 maybe you know how we can update config?)

Oh, I meant updates, for example:

Version 1.0:
```yml
name: WebSK
files: true
```
Version 2.0:
```yml
name: WebSK
files-server: true
config: true
```
(I removed files and I added files-server and config)

And how we can change values to add/remove elements in config without clearing data? I meant something like creating migrations. Now I only added simple warning at line 53

ItsTheSky commented:
You can use / create a sort of getOrSetDefault method (if the specified key doesn't exists, it will sets it to the default provided value)

Norbiros commented:
So we can create file like `migrations/Migration01.java` and it should detect values specific for version (for example `files`) and if it finds that it runs code that saves current config to memory, loads new one and replaces data with data in memory. You can also create renaming rules and things like that. I can start work on that :)

But how you fix that problem in DiSky?

Norbiros commented:
Maybe we can add `config-version` to config?

ItsTheSky commented:
As you wish, both way are usable & should works

ItsTheSky commented:
Should I merge this PR ?

Norbiros committed:
Add unsupported version warning.

ItsTheSky committed:
:rocket: Now using Skript to parse expressions

Norbiros committed:
fix: Paper 1.21
